project.clj

```diff
@@ -47,6 +47,7 @@
                       [clj-time                                  "0.15.1"]
                       [com.github.grinnbearit/freemarker-clj     "-SNAPSHOT"]
                       [irresponsible/tentacles                   "0.6.2"]
```

Comment:
You'll need to add an exclusion for `pathetic` here, to avoid conflicting versions.